packages/office-addin-manifest/test/manifests/manifest.json

```diff
@@ -1,120 +1,82 @@
-            "pinnable": false
-          }
-        ]
-      },
-      {
-        "id": "ExecuteFunction",
-        "type": "general",
-        "code": {
-          "page": "https://localhost:3000/commands.html",
-          "script": "https://localhost:3000/commands.js"
-        },
-        "lifetime": "short",
-        "actions": [
-          {
-            "id": "ExecuteFunction.action",
-            "type": "execution",
-            "name": "action"
-          }
-        ]
-      }
-    ],
-    "ribbons": [
-      {
-        "contexts": [ "MessageReadCommandSurface" ],
-        "tabs": [
-          {
-            "id": "TabDefault",
-            "groups": [
+  "authorization": {
+      "permissions": {
+          "resourceSpecific": [
               {
-                "id": "msgReadGroup",
-                "label": "Contoso Add-in",
-                "icons": [
-                  { "size": 16, "file": "" },
-                  { "size": 32, "file": "" },
-                  { "size": 80, "file": "" }
-                ],
-                "controls": [
-                  {
-                    "id": "msgReadOpenPaneButton",
-                    "type": "Button",
-                    "label": "Show Taskpane",
-                    "icons": [
-                      { "size": 16, "file": "https://localhost:3000/assets/icon-16.png" },
-                      { "size": 32, "file": "https://localhost:3000/assets/icon-32.png" },
-                      { "size": 80, "file": "https://localhost:3000/assets/icon-80.png" }
-                    ],
-                    "supertip": {
-                      "title": "Show Taskpane",
-                      "description": "Opens a pane displaying all available properties."
-                    },
-                    "action": "ShowTaskpane.show"
-                  },
-                  {
-                    "id": "ActionButton",
-                    "type": "Button",
-                    "label": "Perform an action",
-                    "icons": [
-                      { "size": 16, "file": "https://localhost:3000/assets/icon-16.png" },
-                      { "size": 32, "file": "https://localhost:3000/assets/icon-32.png" },
-                      { "size": 80, "file": "https://localhost:3000/assets/icon-80.png" }
-                    ],
-                    "supertip": {
-                      "title": "Perform an action",
-                      "description": "Perform an action when clicked."
-                    },
-                    "action": "ExecuteFunction.action"
-                  }
-                ]
+                "name": "Mailbox.ReadWrite",
+                "type": "Delegated"
               }
-            ]
-          }
-        ]
+          ]
+      }
+  },
+  "validDomains": [
+    "contoso.com"
+  ],
+  "extension": {
+      "requirements": {
+          "scopes": [
+            "mail"
+          ],
+          "capabilities": [
+             { "name": "AddinCommands", "minVersion": "1.1" }
```

Comment:
Do we not need Mailbox?  Why are we Adding "AddinCommands"?

Reply:
Do we? I'm not sure. As said previously, I am mostly basing myself on the Alpha example.
It doesn't define Mailbox here, it just on `runtime` later on.

Reply:
Capabilities is an array of requirements . . . I wouldn't think the names of those are tied directly to the schema since it really depends on the host support vs. the add-in requirements and should be independent from the schema.

Reply:
Sure, I will add the "Mailbox" as a dependency